securitycenter/docs/snippets_findings.py

```diff
@@ -0,0 +1,603 @@
+#!/usr/bin/env python
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Examples of working with source and findings in Cloud Security Command Center."""
+
+from itertools import chain
+import os
+import pytest
+
+
+@pytest.fixture(scope="module")
+def organization_id():
+    """Get Organization ID from the environment variable """
+    return os.environ["GCLOUD_ORGANIZATION"]
+
+
+@pytest.fixture(scope="module")
+def source_name(organization_id):
+    from google.cloud import securitycenter as securitycenter
```

Comment:
Just "from google.cloud import securitycenter" is enough.

Reply:
done.

Reply:
done.